deb/openmediavault/usr/share/openmediavault/datamodels/conf.system.notification.email.json

```diff
@@ -21,8 +21,8 @@
 			"default": 25
 		},
 		"tls": {
-		    "type": "boolean",
-			"default": false
+		    "type": "string",
```

Comment:
Please use a string enum which limits the possible values to the minimum.
Example:
```
"execution":{
			"type": "string",
			"enum": [
				"exactly",
				"hourly",
				"daily",
				"weekly",
				"monthly",
				"yearly",
				"reboot"
			],
			"default": "exactly"
		},
```